docs/rest_api.rst

```diff
@@ -639,27 +672,145 @@ be rejected.
      HTTP/1.1 200 OK
      Content-Type: application/json
 
-     [
-         {
-             "event_type": "ChannelNewBalance",
-             "participant": "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
-             "balance": 150000,
-             "block_number": 54388
-         }, {
-             "event_type": "TransferUpdated",
-             "token_address": "0x91337A300e0361BDDb2e377DD4e88CCB7796663D",
-             "channel_manager_address": "0xC0ea08A2d404d3172d2AdD29A45be56dA40e2949"
-         }, {
-             "event_type": "EventTransferSentSuccess",
-             "identifier": 14909067296492875713,
-             "block_number": 2226,
-             "amount": 7,
-             "target": "0xc7262f1447FCB2f75AB14B2A28DeEd6006eEA95B"
-         }
-     ]
+    [
+        {
+            args: {
+                channel_identifier: "0xa152038763d73b05df7b036f477236b527ad14a249e4077fb4048d845226ac43",
+                participant1_amount: 60,
+                participant2_amount: 955
+            },
+            event: "ChannelSettled",
+            logIndex: 3,
+            transactionIndex: 2,
+            transactionHash: "0x6460c83a8005b3c04179e4d63817d7c5877e0d3ed514a60b25cfd8368330d7f4",
+            address: "0x7692624c0C43285D51c0DdDfb2A6388f5002762E",
+            blockHash: "0x988abf5d672a9d6e07a2210ca5f4391bf9ef38e65896457f1bf95024d4f172f0",
+            block_number: 3694882
+        }, ...
+    ]
+    
+    
+  **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /blockchain_events/payment_networks/0x0f114A1E9Db192502E7856309cc899952b3db1ED/channels/0x82641569b2062B545431cF6D7F0A418582865ba7 HTTP/1.1
+      Host: localhost:5001
 
+  **Example Response**:
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+    [
+        {            
+            args: {
+                channel_identifier: "0xa152038763d73b05df7b036f477236b527ad14a249e4077fb4048d845226ac43",
+                participant1: "0x82641569b2062B545431cF6D7F0A418582865ba7",
+                participant2: "0x8A0cE8bDA200D64d858957080bf7eDDD3371135F",
+                settle_timeout: 600
+            },
+            event: "ChannelOpened",
+            logIndex: 1,
+            transactionIndex: 1,
+            transactionHash: "0x6c1ea37c2d401244e336863a2c7e149a2b364655b6423af472e48044878db107",
+            address: "0x7692624c0C43285D51c0DdDfb2A6388f5002762E",
+            blockHash: "0xcc88f89b3278b7abcb49eeabfe055df5de60449eac5876e77ebf0fcb7ab968f8",
+            block_number: 3694177
+        }, ...
+    ]
+    
+    
   :statuscode 200: For successful query
   :statuscode 400: If the provided query string is malformed
-  :statuscode 404: If the channel does not exist
+  :statuscode 404: If the token_address does not exist
```

Comment:
What happens if the token address exists but we have no channel with the partner? Also a 404?